README.adoc

```diff
@@ -72,7 +72,7 @@ Relevant::
   Relevance also includes ensuring that the goals are aligned with current industry standards, technologies, and practices to prepare learners for real-world challenges.
 
 Authoritative:: 
-  Learning goals should be based on established content, industry best practices, and proven research. 
+  Learning goals should be <<handling-references, based on established content>> industry best practices, and proven research. 
```

Comment:
should it be <<handling-references>> instead of <<handling-references, based on established content>>?

Reply:
it is correct to write <<a,b>> for AsciiDoc cross references, see https://docs.asciidoctor.org/asciidoc/latest/macros/xref/